lib/commands/description.js

```diff
@@ -71,8 +76,44 @@ async function main() {
             process.exit(1);
         }
 
+        let backupFile = filePath;
+        if (Program.backup) {
+            backupFile = `${backupFile}~`;
+            await fs.writeFile(backupFile, info.description);
+            console.log('Backup written to ', backupFile);
+        }
+
         // Read the description file
         const description = await fs.readFile(filePath, 'utf-8');
+
+        if (Program.patch) {
+            if (info.description === description) {
+                // Equal contents.
+                console.log('No changes');
+            } else {
+                const oldName = backupFile;
+                const newName = filePath;
+
+                const unifiedDiff = jsdiff.createTwoFilesPatch(
+                    oldName,
+                    newName,
+                    info.description,
+                    description
+                );
+                console.log(unifiedDiff);
+                const confirmed = await co(function* () {
+                    const ok = yield confirm('Upload changes? (y/n) ');
+                    process.stdin.pause();
```

Comment:
This seems unnecessary.

Reply:
I meant `process.stdin.pause()` (which is also the line I commented on 😉). The docs of `co-prompt` don't mention `process.stdin.pause()`, so I assume it's not necessary.

Reply:
`process.stdin.pause()` is necessary, though, I just verified it ;) I don't know why co-prompt would behave so unexpectedly, but it does keep reading from stdin, otherwise. And that keeps the process alive for longer than it should.